src/dotenv/main.py

```diff
@@ -7,16 +7,17 @@
 import shutil
 import sys
 import tempfile
-from typing import (Dict, Iterator, List, Match, Optional,  # noqa
-                    Pattern, Union, TYPE_CHECKING, Text, IO, Tuple)
 import warnings
 from collections import OrderedDict
 from contextlib import contextmanager
 
-from .compat import StringIO, PY2, to_env
+from .compat import StringIO, PY2, to_env, IS_TYPE_CHECKING
 from .parser import parse_stream
 
-if TYPE_CHECKING:  # pragma: no cover
+if IS_TYPE_CHECKING:  # pragma: no cover
```

Comment:
I think the pragma is probably unnecessary since `if IS_TYPE_CHECKING` is already marked as "to be ignored" in `setup.cfg`.

Reply:
yeah I think I likely added it because the coverage engine was angry about this line anyway, removed